llama_stack/providers/adapters/inference/bedrock/bedrock.py

```diff
@@ -0,0 +1,359 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.

```

Comment:
could you install `pre-commit` and run it on the commit?

Reply:
Will do.